ui/components/upload/disclaimer.tsx

```diff
@@ -7,46 +7,71 @@ import {
   AlertDialogFooter,
   AlertDialogHeader,
   AlertDialogTitle,
-} from "@/components/ui/alert-dialog"

```

Comment:
This `"` to `'` is something my code editor did. That reminds me, maybe the project should have a something like a prettier installed? Do you prefer I raise a PR for that?